Include locations for release notifications in the readme.

rolfyone opened the pull request:
fixes #4266

Signed-off-by: Paul Harris <paul.harris@consensys.net>


## Documentation

- [X] I thought about documentation and added the `documentation` label to this PR if updates are required.

## Changelog

- [X] I thought about adding a changelog entry, and added one if I deemed necessary.

rolfyone committed:
Include locations for release notifications in the readme.

fixes #4266

Signed-off-by: Paul Harris <paul.harris@consensys.net>

ajsutton commented on README.md:
```diff
@@ -38,6 +38,12 @@ or [zip format](https://artifacts.consensys.net/public/teku/raw/names/teku.zip/v
 
+Release notifications are available via:
+* `teku` in [Consensys Discord](https://discord.gg/7hPv2T6), 
+* `client-updates` in [Ethstaker Discord](https://discord.gg/u2q7FRv)
```
I probably wouldn't list this one as an official source. It is useful but we don't actually control it so can't do anything if they stop including our announcements there in the future.

ajsutton commented on README.md:
```diff
@@ -38,6 +38,12 @@ or [zip format](https://artifacts.consensys.net/public/teku/raw/names/teku.zip/v
+* `client-updates` in [Ethstaker Discord](https://discord.gg/u2q7FRv)
+* Subscribe to release notifications on github for [teku](https://github.com/ConsenSys/teku)
+* Follow us on [Twitter](https://twitter.com/Teku_ConsenSys)
+* Sign up to our [release announcements](https://pages.consensys.net/teku-sign-up) email list (release and important announcements only, no marketing)
```
Probably worth listing the email list first, then twitter, then our discord and finally GitHub release announcements. Probably arbitrary but feels like it puts the more reliable but less discoverable channels first.

rolfyone committed:
Increase the seen attestation cache size.

The previous cache size was too small to cover the number of attestations being gossiped if all networks were subscribed to.

fixes #4266

Signed-off-by: Paul Harris <paul.harris@consensys.net>

rolfyone committed:
review feedback

Signed-off-by: Paul Harris <paul.harris@consensys.net>

rolfyone committed:
Merge remote-tracking branch 'upstream/master' into 4266

ajsutton reviewed (approved):
LGTM.

ajsutton commented on util/src/main/java/tech/pegasys/teku/util/config/Constants.java:
```diff
@@ -116,7 +116,7 @@
   // Teku Networking Specific
   public static final int VALID_BLOCK_SET_SIZE = 1000;
-  public static final int VALID_ATTESTATION_SET_SIZE = 1000;
+  public static final int VALID_ATTESTATION_SET_SIZE = 21000;
```
I take that back, this appears to have been pushed to the wrong PR. :)

rolfyone replied:
definitely the wrong PR...

ajsutton reviewed (changes requested):
See stray change comment.

rolfyone committed:
accidental commit, removing change from this PR.

Signed-off-by: Paul Harris <paul.harris@consensys.net>

ajsutton reviewed (approved):
Now it looks good. :)
